packages/airnode-validator/README.md

```diff
@@ -66,7 +73,8 @@ validator templates with conversion actions.
 
 ## Usage
 
-Convertor works the same way as validator and can be invoked with the `convert` command, for example:
+Convertor works the same way as validator (including interpolation) and can be invoked with the `convert` command, for
+example:
 
 ```sh
 yarn run convert --template="conversions/oas@3.0------ois@0.2.json" --specs="exampleSpecs/OAS.specs.json"
```

Comment:
Can you change this command as well?

I think the validator should always depend on both `secrets.env` and `config.json` and throw an error if one is not provbided.

Reply:
I think that we agreed on the call that if the secrets file is not provided the interpolation is simply skipped.

Reply:
I don't recall. 

Anyway, is that a good idea? This can cause problems for the user with no real benefit.

Reply:
Not all validated files contain sensitive information and don't need an interpolation (e.g. `receipt.json`). You would have to provide an empty file just to make the flow work. That doesn't make much sense to me. And I don't really see problems it would cause, can you elaborate?

Reply:
This sounds like not very intuitive behavior, it will be very confusing and cause many problems. From what I've understood the longterm goal is to limit the work with config files needed to run airnode to as little as possible, especially these kind of checks should run automatically in the background without user knowing. There is already github action set up in one of the repositories that's validating the OIS files in the repository. Having to provide valid `secrets.env` file in order to validate the OIS just does not make much sense, and it's much easier to just put one extra argument when the interpolation is needed rather than creating an empty file every single time when it's not.

Reply:
> Not all validated files contain sensitive information and don't need an interpolation

Good point. But I was mainly talking about validating `config.json` since interpolations is not supported (and doesn't make sense) for `receipt.json`. I'd like to have `secrets.env` required when validting `config.json` - because it;'s easy to forget to supply the file and accept invalid configuration. 

Another option is to issue a warning in this case (when validating config.json without secrets)

Reply:
Having a special behavior just for `config.json` brings another layer to the confusion imho. From the perspective of the user I still think its going to be much easier to figure out that I forgot to provide `.env` file when its needed (just by looking at the values that are in the error messages), rather then accidentally validating with `.env` file that I didn't even know Im validating with.

Reply:
I disagree actually - `config.json` on its own doesn't make much sense. We should always be talking a about the pair (config.json + secrets.env) as this form "the airnode configuration". Also, most people will be using the secrets as well because the providerUrl is a secret and you need that for every Airnode deployment.

That said, I don't wanna block this PR because of this - so feel free to merge and we can implement the check for secrets separately.

Reply:
I agree with Jakub in this in general (interpolation should be optional and explicit)